packages/utils/odsp-doclib-utils/package.json

```diff
@@ -13,6 +13,36 @@
 	"sideEffects": false,
 	"exports": {
 		".": {
+			"import": {
+				"types": "./lib/odsp-doclib-utils-public.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/odsp-doclib-utils-public.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./beta": {
+			"import": {
+				"types": "./lib/odsp-doclib-utils-beta.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/odsp-doclib-utils-beta.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./fruit": {
```

Comment:
What is meant by fruit? 🤔

Reply:
~"These are the APIs of the forbidden fruit. You must not use them."~

I believe it was just random-ish to not use @alpha, which could cause confusion among our partners.